src/client/pages/profile/queries.js

```diff
@@ -15,3 +15,45 @@ export const GET_PROFILE = gql`
     }
   }
 `;
+
+export const GET_USER_PROFILE = gql`
```

Comment:
Why do we need to have separate queries: `GET_USER_PROFILE` and  `GET_PUBLIC_PROFILE`. Can we use a single query for both situations?

Reply:
`GET_USER_PROFILE` finds the logged in user by email. Remember, upon login, the user's email was returned and used to fetch the user's profile after they were redirected to the profile page. `GET_PUBLIC_PROFILE` finds another user's profile by id. I suppose we could modify several files just get rid of one query, but at this point with the deadline so close, I'd rather put that off until later. Is that ok for now?

Reply:
Update: I have modified the code as requested to have only one profile query `GET_USER_PROFILE`.

As the deadline for the project is almost here and this PR has not received much attention, I'm going to approve and merge it.